### DISCUSSION TITLE
Find the hidden country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#512] [Galaxy Trivia] [answered]
If a Dalmatian can catch four balls in four minutes, can a Dalmatian catch two balls in two minutes?

*answer using country flag emoji*

**Comment by peterbaccaro** [comment]
:canada: - **can a Da**

**Comment by thuongtruong1009** [comment, marked as answer]
🇳🇿

The hidden country is New Zealand.

![image](https://user-images.githubusercontent.com/71834167/230750102-03e41f57-c62e-462e-b2c5-cfbb347b7fc2.png)
